Update app-sdk-ios.md (#3800)
MicrosoftDocs/memdocs: memdocs/intune/developer/app-sdk-ios.md

4. Ensure the steps to give your iOS app permissions to the app protection policy (APP) service are followed. Use the instructions in the [getting started with the Intune SDK guide](app-sdk-get-started.md#next-steps-after-integration) under "[Give your app access to the Intune app protection service (optional)](app-sdk-get-started.md#give-your-app-access-to-the-intune-app-protection-service-optional)".  

> [!NOTE]

> > If the App Protection Policy is related to Managed Devices, it is necessary to create an App Configuration Profile of the application that has Intune integrated.
> 
> 
> > [!NOTE]

Reviewer: ```suggestion
> If the app protection policy is related to managed devices, creating an app configuration profile of the application that has Intune integrated is also necessary.
>
```
